pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -101,3 +105,21 @@ def test_write_append_mode(ext, mode, expected):
 
         for index, cell_value in enumerate(expected):
             assert wb2.worksheets[index]["A1"].value == cell_value
+
+
+@pytest.mark.xfail(openpyxl.__version__ > "3.0.1", reason="broken change in openpyxl")
+def test_to_excel_with_openpyxl_engine(ext, tmpdir):
+    # GH 29854
+    # TODO: Fix this once newer version of openpyxl fixes the bug
```

Comment:
@charlesdong1991 any idea if this TODO can be addressed?

Reply:
this `xfail` should be removed!! new version of `openpyxl` has fixed the issue @jbrockmendel

Reply:
emm, seems have been removed on master.